File: 2013/E.java
package a2013;

import java.util.Scanner;

public class E {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		while(sc.hasNext()) {
			int cont=0;
			int mergulharam = sc.nextInt();
			int voltaram = sc.nextInt();
			
			
			int[] mergulhadores = new int[mergulharam]; 
			int[] numerosQueVoltaram = new int[voltaram];
			
			for(int i=0; i<mergulhadores.length; i++) {
				mergulhadores[i] = i+1;
			}
			
			for(int j=0; j<numerosQueVoltaram.length; j++) {
				numerosQueVoltaram[j] = sc.nextInt();
			}

			for(int k=0; k<mergulhadores.length; k++) {
				cont = 0;
				for(int l=0; l<numerosQueVoltaram.length; l++) {
					if(mergulhadores[k] != numerosQueVoltaram[l]) {
						cont++;
						if(cont == numerosQueVoltaram.length) {
							System.out.println(mergulhadores[k]);
						}
					}
				}
			}
			if(mergulharam == voltaram) {
				System.out.println("*");
			}
		}
	}
}

File: 2013/E.cpp
#include <iostream>
using namespace std;

int main(){
	while(true){
  	
  		int cont=0;
		int mergulharam;
		int voltaram;
	
		cin >> mergulharam >> voltaram;
	
		int mergulhadores[mergulharam];
		int numerosQueVoltaram[voltaram];
	
		for(int i=0; i<mergulharam; i++){
			mergulhadores[i] = i+1;
		}
	
		for(int j=0; j<voltaram; j++){
			cin >> numerosQueVoltaram[j];
		}
	
		for(int k=0; k<mergulharam; k++){
			cont = 0;
			for(int l=0; l<voltaram; l++){
				if(mergulhadores[k] != numerosQueVoltaram[l]){
					cont++;
					if(cont == voltaram){
						cout << mergulhadores[k] << " ";
					}
				}
			}
		}
		if(mergulharam == voltaram){
			cout << "*" << endl;
		}
	}
}
